BurnerB/FreeMentors-V2
# [#167961853]Structure app,add dependancies and intergrations

#### What does this PR do?
Set up the app
#### Description of Task to be completed?
* Structure the app to enable separation of concerns and future scalability.
* Set up dependancies such as eslint and babel.
* Intergrate continuous interation and test coverage
#### How should this be manually tested?
* Clone this  repository https://github.com/BurnerB/FreeMentors-V2.git and cd into FreeMentors-V2
* checkout to branch ch-Structure-dependancies-intergrations-167961853 
* Run npm --init  to install all dependancies.
* Run command npm start on your terminal
* visit http://127.0.0.1:5000/ on any browser
* You should see a message `Hello world!! Your app is working`

#### What are the relevant pivotal tracker stories?
[#167961853](https://www.pivotaltracker.com/story/show/167961853)
#### Screenshots (if appropriate) 
![working](https://user-images.githubusercontent.com/45785786/63257192-e6c8cd80-c281-11e9-9b9e-1e835e2284fa.png)
